test/test.js

```diff
@@ -1,19 +1,19 @@
-import test from 'ava';
-import stripAnsi from 'strip-ansi';
-import ansiEscapes from 'ansi-escapes';
-import chalk from 'chalk';
-import defaultFixture from './fixtures/default.json';
-import noLineNumbers from './fixtures/no-line-numbers.json';
-import lineNumbers from './fixtures/line-numbers.json';
-import sortOrder from './fixtures/sort-by-severity-then-line-then-column.json';
-import messages from './fixtures/messages.json';
-import data from './fixtures/data.json';
-import eslintFormatterPretty from '..';
-
-const severityFilter = desiredSeverity => ({severity}) => severity === desiredSeverity;
+'use strict';
+
+const test = require('ava');
+const stripAnsi = require('strip-ansi');
+const ansiEscapes = require('ansi-escapes');
+const chalk = require('chalk');
+const defaultFixture = require('./fixtures/default.json');
+const noLineNumbers = require('./fixtures/no-line-numbers.json');
+const lineNumbers = require('./fixtures/line-numbers.json');
+const sortOrder = require('./fixtures/sort-by-severity-then-line-then-column.json');
+const messages = require('./fixtures/messages.json');
+const data = require('./fixtures/data.json');
+const eslintFormatterPretty = require('..');
```

Comment:
I'm holding off on upgrading AVA in my packages. Too much churn. I'll be able to target native ESM soon anyway. You can upgrade AVA to `^2.4.0` instead.

Reply:
When I install `ava@^2.4.0` it causes `xo` to complain about 'ava/no-import-test-files' for all the `./fixture/*.json` imports.  Any preference for dealing with this:
* Disable the rule for the test file
* Move `./test/fixtures` to `./fixtures`
* Some other option?

Reply:
> Disable the rule for the test file

👍

Reply:
Done, thanks for the feedback.